Index: app/src/test/java/com/revature/models/UserTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.revature.models;\r\n\r\nimport junit.framework.TestCase;\r\n\r\npublic class UserTest extends TestCase {\r\n\r\n    public void testGetUserRole() {\r\n        User user = new User(\"test\", \"user\", UserRole.EMPLOYEE, \"email\", \"password\");\r\n        \r\n    }\r\n\r\n    public void testSetUserRole() {\r\n    }\r\n\r\n    public void testTestToString() {\r\n    }\r\n\r\n    public void testGetId() {\r\n    }\r\n\r\n    public void testSetId() {\r\n    }\r\n\r\n    public void testGetF_name() {\r\n    }\r\n\r\n    public void testSetF_name() {\r\n    }\r\n\r\n    public void testGetL_name() {\r\n    }\r\n\r\n    public void testSetL_name() {\r\n    }\r\n\r\n    public void testGetEmail() {\r\n    }\r\n\r\n    public void testSetEmail() {\r\n    }\r\n\r\n    public void testGetPassword() {\r\n    }\r\n\r\n    public void testSetPassword() {\r\n    }\r\n}
===================================================================
diff --git a/app/src/test/java/com/revature/models/UserTest.java b/app/src/test/java/com/revature/models/UserTest.java
--- a/app/src/test/java/com/revature/models/UserTest.java	
+++ b/app/src/test/java/com/revature/models/UserTest.java	
@@ -5,7 +5,7 @@
 public class UserTest extends TestCase {
 
     public void testGetUserRole() {
-        User user = new User("test", "user", UserRole.EMPLOYEE, "email", "password");
+        User user = new User("james", "smith", UserRole.EMPLOYEE, "james.smith@gmail.com", "password");
         
     }
 
