Index: Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>FROM openjdk:8-jdk-alpine\r\nCOPY app/build/libs/app.jar project1.jar\r\nCMD [\"java\", \"-jar\", \"project1.jar\"]\r\n
===================================================================
diff --git a/Dockerfile b/Dockerfile
--- a/Dockerfile	
+++ b/Dockerfile	
@@ -1,3 +1,3 @@
-FROM openjdk:8-jdk-alpine
-COPY app/build/libs/app.jar project1.jar
-CMD ["java", "-jar", "project1.jar"]
+FROM openjdk:11
+COPY app/build/libs/app.jar demo.jar
+CMD ["java", "-jar", "demo.jar"]
\ No newline at end of file
